contracts/hydro/src/query.rs

```diff
@@ -189,3 +207,13 @@ pub struct ValidatorPowerRatioResponse {
 pub struct ICQManagersResponse {
     pub managers: Vec<Addr>,
 }
+
+#[cw_serde]
+pub struct LiquidityDeploymentResponse {
+    pub liquidity_deployment: LiquidityDeployment,
+}
+
+#[cw_serde]
+pub struct RoundTrancheLiquidityDeploymentsResponse {
```

Comment:
These structs should be added to `schema.rs` to get the JSON/TS schema generated.